_locales/en.xml

```diff
@@ -678,6 +678,8 @@
     <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the Inbox</string>
     <string name="pref_only_fetch_mvbox_title">Only Fetch from DeltaChat Folder</string>
     <string name="pref_only_fetch_mvbox_explain">Ignore other folders. Requires your server to move chat messages to the DeltaChat folder.</string>
+    <string name="pref_disable_idle_title">Disable IMAP IDLE</string>
+    <string name="pref_disable_idle_explain">Do not use IMAP IDLE extension even if the server supports it. Enabling this option will delay message retrieval, enable it only for testing.</string>
```

Comment:
please don't add stuff directly here it will get lost, It needs to be added to strings.xml in the android repo, if you need it now add it to the "untranslated json"-file in the same folder.